pyseir/cli.py

```diff
@@ -48,6 +49,26 @@ def _generate_whitelist():
     gen.generate_whitelist()
 
 
+def _generate_infection_rate_metric(regions: List[pipeline.Region] = None):
+    """
+    Apply infer_rt.run_rt_for_fips for each region in regions and return a combined results table
+    with a unique row for each region+date combination.
+    """
+    if regions is None:
+        regions = combined_datasets.load_us_latest_dataset().all_fips
+
+    with Pool(maxtasksperchild=10) as p:
+        results = p.map(infer_rt.run_rt_for_fips, regions)
+
+    def label_data(label, data):
+        """Modify the returned dataframe to meet requirements"""
+        data = data.reset_index(drop=False)  # Move date column from index to columns
+        data["fips"] = label  # Add region column id
+        return data
+
+    rt_df = pd.concat([label_data(region, data) for region, data in zip(regions, results)])
```

Comment:
does it handle none appropriately?

Reply:
Good point. I'll have it return an empty dataframe if results evaluates false.